chart/etcd-backup-restore/templates/etcd-bootstrap-configmap.yaml

```diff
@@ -128,3 +137,67 @@ data:
       # Path to the client server TLS trusted CA cert file.
       trusted-ca-file: /var/etcd/ssl/ca/ca.crt
 {{- end }}
+
+  etcdbr.conf.yaml: |-
+    # etcd-connection-config
+    etcdConnectionConfig:
+      connectionTimeout: "{{ .Values.backup.etcdConnectionTimeout }}"
+{{- if and .Values.etcdAuth.username .Values.etcdAuth.password }}
+      username: "{{ .Values.etcdAuth.username }}"
+      password: "{{ .Values.etcdAuth.password }}"
+{{- end }}
+{{- if .Values.etcdTLS }}
+      endpoints:
+        - "https://localhost:{{ .Values.servicePorts.client }}"
```

Comment:
Verify if localhost will work? This field depends on the `san` fields in the certificate.